server/http.go

```diff
@@ -325,6 +325,10 @@ func (p *Plugin) postMeeting(creator *model.User, meetingID int, channelID strin
 			"meeting_provider":         zoomProviderName,
 		},
 	}
+	if parentID != "" {
```

Comment:
There is no need for this conditional. You can directly add the line `RootID: rootID` to the post definition just before this.

Reply:
It doesn't work when there is no valid `RootId`.(If user schedule meeting by clicking zoom button from channel header, I had to pass `""` as substitute).
It fails with Error: `Invalid value for RootID`

Reply:
That seems weird. If you click form the channel header, root ID should not be available, and therefore be blank (`""`). Golang values also have defaults, and this being a string, the default is `""`. So it should not change the behavior between not doing anything, and assigning `""`.

Can you make sure what is coming as RootID from the channel header call?

Reply:
Sure